go/vt/vtgate/vindexes/vschema.go

```diff
@@ -1147,6 +1153,14 @@ func (vschema *VSchema) FindRoutedShard(keyspace, shard string) (string, error)
 	return keyspace, nil
 }
 
+func (vschema *VSchema) GetCreated() time.Time {
+	return vschema.created
+}
+
+func (vschema *VSchema) ResetCreated() {
+	vschema.created = time.Time{}
+}
+
```

Comment:
nit: public methods would need a comment.
`ResetCreated` is only used in tests. is this method needed?

Reply:
There are tests which compare the vschema protos, which fail without this, since the `got` vschema will have a non-deterministic created time. It is public because it is also used in the vschema manager in vtgate.